app/src/main/java/edu/wisc/ece/pinpoint/utils/FirebaseDriver.java

```diff
@@ -157,6 +168,12 @@ public void handleNewUser() {
         db.collection("private").document(uid).set(data).addOnSuccessListener(
                         t -> Log.d(TAG, String.format("Wallet for user %s created!", uid)))
                 .addOnFailureListener(e -> Log.w(TAG, "Error creating wallet document", e));
+
+        // create follower and following maps for new user
+        HashMap<String, Object> socials = new HashMap<>();
+        socials.put("followers", null);
+        socials.put("following", null);
+        db.collection("social").document(uid).set(socials);
```

Comment:
Does this push list types to the database? Not sure if passing null will do that so I suggest passing an empty array to ensure the fields get the correct type.

Reply:
Yes, this is the suggested way to push any type to the database. It works with lists as well

Reply:
My concern is with the security rules that check the size of this array. If it isn't a list type in Firestore the `.size()` call may throw an error. Maybe you've tested this though and it works? Here's the type I'm seeing in Firestore: 
![image](https://user-images.githubusercontent.com/55991831/231958503-bc12fda4-7687-4bed-9f84-1af7f9a04a21.png)